components/tidb_query_expr/src/impl_time.rs

```diff
@@ -696,6 +730,75 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_add_sub_datetime_and_duration() {
+        let mut ctx = EvalContext::default();
+        let null_cases = vec![
+            ("", "11:30:45.123456"),
+            ("2019-01-01 01:00:00", ""),
+            ("", ""),
+        ];
+        for (arg1, arg2) in null_cases {
```

Comment:
Maybe we can test the `null_cases` and `normal_cases` at the same time, please take `test_month_name` as ref.

Reply:
Sure~

done